translator/translate/otel/pipeline/applicationsignals/translator_test.go

```diff
@@ -125,7 +125,7 @@ func TestTranslatorMetricsForKubernetes(t *testing.T) {
 			},
 			want: &want{
 				receivers:  []string{"otlp/application_signals"},
-				processors: []string{"resourcedetection", "awsapplicationsignals"},
+				processors: []string{"metricstransform/application_signals", "resourcedetection", "awsapplicationsignals"},
```

Comment:
nit: I know it's unrelated, but could we add `application_signals` to the `resourcedetection` processor as well? Could be done in a separate PR if you'd like.

Reply:
I will do in a follow-up PR.